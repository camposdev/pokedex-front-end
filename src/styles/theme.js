const theme = {
  palette: {
    red: '#fa4242',
    redHover: '#c51616',
    black: '#000000',
    black50: 'rgba(0,0,0,0.5)',
    black20: 'rgba(0,0,0,0.2)',
    black10: 'rgba(0,0,0,0.1)',
    white: '#ffffff'
  },
  breakpoints: {
    sm: '576px',
    md: '768px'
  }
}

export default theme
