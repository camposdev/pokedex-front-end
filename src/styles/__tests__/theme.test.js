import theme from '../theme'

test('snapshot theme', () => {
  expect(theme).toMatchSnapshot()
})
