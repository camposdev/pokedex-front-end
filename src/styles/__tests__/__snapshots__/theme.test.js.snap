// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot theme 1`] = `
Object {
  "breakpoints": Object {
    "md": "768px",
    "sm": "576px",
  },
  "palette": Object {
    "black": "#000000",
    "black10": "rgba(0,0,0,0.1)",
    "black20": "rgba(0,0,0,0.2)",
    "red": "#FF5656",
    "white": "#ffffff",
  },
}
`;
